config/locales/en/admin/whats_new.yml

```
@@ -25,7 +25,12 @@ en:
       recent_changes:
         heading: Recent changes
         updates:
-          - heading: New images 
+         - heading: New call for evidence content type
+            area: Creating and updating documents
+            type: new
+            date: 9 August 2023
+            body_govspeak: |
+              We have released a call for evidence content type in Whitehall. This is similar to consultations, but is now available in the new document menu. Outcomes are optional, but for details [read more in the GOV.UK guidance](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/calls-for-evidence).
```

I think you have added an extra space here that's making the tests fail. I think this should fix it :)
```suggestion
         - heading: New call for evidence content type
           area: Creating and updating documents
           type: new
           date: 9 August 2023
           body_govspeak: |
              We have released a call for evidence content type in Whitehall. This is similar to consultations, but is now available in the new document menu. Outcomes are optional, but for details [read more in the GOV.UK guidance](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/calls-for-evidence).
```